src/Containers/FileUpload/FileUpload.tsx

```diff
@@ -488,39 +489,28 @@ export const FileUpload: React.FC<IFileUploadProps> = ({
         setIsSuccess(false)
         setIsFailure(false)
         acceptedFiles.forEach((file) => {
-            const reader = new FileReader();
-            reader.onload = () => {
-                if (reader.result) {
-                    let base64StringFile = '';
-                    if (typeof reader.result === 'string') {
-                        base64StringFile = btoa(reader.result);
-                    } else {
-                        const bytes = Array.from(new Uint8Array(reader.result));
-                        base64StringFile = btoa(
-                            bytes
-                                .map((item) => String.fromCharCode(item))
-                                .join(''),
-                        );
-                    }
-                    if (base64StringFile) {
-                        base64StringFileRef.current=base64StringFile
-                        setIsSuccess(true)
-                        setIsFailure(false)
-                        setIsUploading(false)
-                        setTimeout(()=>{
-                            if(isMounted.current){
-                                setIsSuccess(false)
-                            }
-                        },messageDuration)
-                    }
+            const worker=new Worker('./worker.js', { type: 'module' })
+            workerRef.current=worker
+            worker.postMessage({file})
+            worker.onmessage=(e)=>{
+                const {base64StringFile}=e.data
+                if(base64StringFile!==undefined){
```

Comment:
What if it equals null?
Use 
!Var

Reply:
but what if is an empty string because file read was empty?

Reply:
maybe and can use ??

Reply:
I don't think it can be null... can be undefined or empty string (or normal string)